ci-operator/step-registry/multiarch/nodes-readiness/multiarch-nodes-readiness-commands.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [ -z "${CONTROL_ARCH}" ] || [ -z "${COMPUTE_ARCH}" ]; then
+  echo "[INFO] CONTROL_ARCH and COMPUTE_ARCH are not set. Skipping this step."
+  exit 0
+fi
+
+if [ "${OCP_ARCH}" == "arm64" ]; then
+  # There's an assumption for which OCP_ARCH will take precedence over the control and compute arches
```

Comment:
On gcp, sounds like `CONTROL_ARCH` and `COMPUTE_ARCH` has higher precedence than `OCP_ARCH`.

Reply:
How about this:
```
control_plane_arch="${CONTROL_ARCH:-${OCP_ARCH}}"
compute_arch="${COMPUTE_ARCH:-${OCP_ARCH}}"
if [ -n "${control_plane_arch}" ] && [ -n "${compute_arch}" ] && [[ "${compute_arch}" != "${control_plane_arch}" ]];
    echo "[INFO] This is not a multi-arch cluster. Skipping this step."
    exit 0
fi
```